src/main/java/teammates/ui/controller/InstructorCourseDetailsPageData.java

```diff
@@ -52,20 +52,20 @@ public void init(InstructorAttributes currentInstructor, CourseDetailsBundle cou
                           + "window.location.href='" + sanitizeForJs(getInstructorCourseRemindLink(courseDetails.course.id)) + "';";
         courseRemindButton = createButton(null, "btn btn-primary", "button_remind", null, 
                                           Const.Tooltips.COURSE_REMIND, "tooltip", onClick, isDisabled);
-        
-        String photoUrl = Const.ActionURIs.STUDENT_PROFILE_PICTURE
-                        + "?" + Const.ParamsNames.STUDENT_EMAIL
-                        + "=%s&" + Const.ParamsNames.COURSE_ID
-                        + "=%s&" + Const.ParamsNames.USER_ID + "=" + account.googleId;
 
         this.sections = new ArrayList<StudentListSectionData>();
+        String studentPhotoUrl = "";
         for (SectionDetailsBundle section: courseDetails.sections) {
             Map<String, String> emailPhotoUrlMapping = new HashMap<String, String>();
             for (TeamDetailsBundle teamDetails : section.teams) {
                 for (StudentAttributes student : teamDetails.students) {
-                    emailPhotoUrlMapping.put(student.email, String.format(photoUrl,
-                                                                          StringHelper.encrypt(student.email),
-                                                                          StringHelper.encrypt(student.course)));
+                    studentPhotoUrl = student.getPublicProfilePictureUrl();
```

Comment:
Can `studentPhotoUrl` be instantiated here instead? I don't think it's used outside the loop.